api/document/js/__tests__/serialize-doc.test.ts

```diff
@@ -53,6 +53,20 @@ describe('serializeDoc()', () => {
     expect(result.title).toEqual('Stufff');
   });
 
+  it('converts footnotes', () => {
+    const node = schema.nodes.para.create({}, [
+      schema.nodes.paraText.create({}, [
+        schema.text('hello'),
+        schema.nodes.inlineFootnote.create({ emblem: '2' }, [
+          schema.text('i am a footnote'),
+        ]),
+      ]),
+    ]);
```

Comment:
If the need to include inlineFootnotes comes up frequently in our tests, we may want to modify the `para` factory to accommodate.